File: 1413/1413.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var minimumValueToGetPositiveStepB = function(root) {
    if (!root) return 0;
    return 1 + Math.max(minimumValueToGetPositiveStepB(root.left), minimumValueToGetPositiveStepB(root.right));
};

File: 1413/1413.go
func minimumValueToGetPositiveStepB(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := minimumValueToGetPositiveStepB(root.Left)
	r := minimumValueToGetPositiveStepB(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}